Reference/Configuration/RuntimeSettings/index.md

```diff
@@ -16,10 +16,15 @@ An example of a configuration could look something like:
   "CMS": {
     "Runtime": {
       "MaxQueryStringLength": 90,
-      "MaxRequestLength": 2048
+      "MaxRequestLength": 2048,
+      "Mode": "BackofficeDevelopment"
     }
   }
 }
 ```
 
 `MaxRequsetLength` is specified in kilobytes, so this configuration would limit requests, and therefore uploaded files, to 2 megabytes, and a maximum query string length of 90 characters.
+
+`Mode` could have three values: `BackofficeDevelopment` (default), `Development` and `Production`
+
+Read more about the details at: [Fundamentals > Setup > Server-Setup > Runtime modes](/documentation/Fundamentals/Setup/Server-Setup/runtime-modes)
```

Comment:
```suggestion
For more information, see the [Runtime modes](/documentation/Fundamentals/Setup/Server-Setup/runtime-modes) article.
```

Reply:
👍😁

Reply:
@idseefeld, 

If you agree with the suggestions, can you click on **Commit Suggestions** -> **Commit changes**. Once done, I'll merge in the PR 😃